examples/progress.js

```diff
@@ -0,0 +1,49 @@
+/* @jsx h */
+const {h, mount, Component, Text, Bar} = require('../');
+
+const MAX = 20;
+
+class ProgressApp extends Component {
+	constructor() {
+		super();
+
+		this.state = {
+			i: 0
+		};
+	}
+
+	render() {
+		const text = `Running `;
```

Comment:
Oops, didn't notice that. Would be nice if the bar just calculated the `left` thing automatically. Feels weird having to manually specify it.